config/default-spinnaker-local.yml

```diff
@@ -110,9 +109,11 @@ providers:
     enabled: ${SPINNAKER_KUBERNETES_ENABLED:false}
 
     primaryCredentials:
-      name: my-docker-registry-account
-      address: https://index.docker.io/
-      repository: library/nginx
+      name: my-docker-registry
```

Comment:
What is the "name"? For all the others, it is the account name within spinnaker -- it has no meaning outside spinnaker. The default ends in "account" to help clarify this. Here, it sounds like name is the registry name, which might be a real name. If so, this isnt standard so perhaps should use a different name. Otherwise, perhaps it should decorate with "account" for consistency.